### DISCUSSION TITLE
Upgrading Opensea SDK to ^2.0.1

### DISCUSSION THREAD

**Discussion by allenchuang** [nawangwe/opensea-whitelabel-website-template#15] [Ideas]
@nawangwe i've created a PR from my fork - you can review here: https://github.com/nawangwe/opensea-whitelabel-website-template/pull/13

**Comment by allenchuang** [comment]
lets chat here so we don't pollute the issues section 😛

**Comment by nawangwe** [comment]
I will have to request an api key to continue development, at the mean time let me figure out any other way to obtain one.
